xbmc/interfaces/json-rpc/VideoLibrary.cpp

```diff
@@ -644,6 +644,8 @@ JSONRPC_STATUS CVideoLibrary::SetSeasonDetails(const std::string &method, ITrans
   std::set<std::string> removedArtwork;
   std::set<std::string> updatedDetails;
   UpdateVideoTag(parameterObject, infos, artwork, removedArtwork, updatedDetails);
+  if (ParameterNotNull(parameterObject, "title"))
+    infos.SetSortTitle(parameterObject["title"].asString());
```

Comment:
Should probably be `SetSortTitle` to be consistent with the code paths

Reply:
This matches the existing implementation from #6041, should I change the way season title works in the rest of the code?

Reply:
Keep it like it is, and let's move this forward this way. We will see if problems appear.
When I get some time I will look at why we're setting this and if this is an oversight.

Then again, this is about setting it from the outside and it might be very confusing for an api, if you try to set `title` and it get's set to `sorttitle`. While setting `sorttitle` will do nothing.

Reply:
I'm up for changing it elsewhere, I wanted to confirm expanding the scope of the PR a bit beyond the JSON-RPC layer before I did.

`sorttitle` is not an option for seasons over JSON-RPC so either way the outside interface is the same, it's just some internal behavior that calls it sort title.

Reply:
If kept like it is then it is possible to get this into v18 (additions to API acceptable under beta, will freeze API at RC) and further changes could then be in another PR for v19. It all depends how quickly this is wanted.

Reply:
I'd say we're good as-is, then.